feat(interview-prep): Converting and Transferring Rosetta problems (#38371)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/lucas-lehmer-test.md

---
id: 5e6dee7749a0b85a3f1fc7d5
title: Lucas-Lehmer test
challengeType: 5
---

## Description
<section id='description'>

> Lucas-Lehmer Test: for $p$ an odd prime, the Mersenne number $2^p-1$ is prime if and only if $2^p-1$ divides $S(p-1)$ where $S(n+1)=(S(n))^2-2$, and $S(1)=4$.

Reviewer: I think the $s need to be removed and replaced with some other tag?

</section>

## Instructions
<section id='instructions'>

> Write a function that returns true if given Mersenne number is prime otherwise returns false.

Reviewer: ```suggestion
Write a function that returns true whether the given Mersenne number is prime or not.
```

Author: @thecodingaviator I think this will make more sense:
Write a function that returns whether the given Mersenne number is prime or not.

Reviewer: Yup. This sounds fine!
